modules/local/transdecoder.nf

```diff
@@ -2,10 +2,10 @@ process TRANSDECODER {
     tag "$meta.id"
     label 'process_medium'
 
-    conda "${moduleDir}/environment.yml"
+    conda "bioconda::transdecoder=5.7.1"
```

Comment:
This looks like a reversal to older syntax. I'm missing something I think.

Reply:
Yes it is, but it's because this is a local module so we don't have a moduleDir